common/changes/@bentley/eslint-plugin/eslint-peerdep_2021-06-18-14-23.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/eslint-plugin",
+      "comment": "",
```

Comment:
Can you add a note to the changelog for this? That way people will know what is needed.